### DISCUSSION TITLE
Umi 3.5 升级到 Umi 4 ,connect 遇到的问题

### DISCUSSION THREAD

**Discussion by xhua007** [umijs/umi#9913] [General]
Unhandled Rejection (TypeError): (0 , umi__WEBPACK_IMPORTED_MODULE_1__.connect) is not a function， 有啥解决的办法吗?

**Comment by fz6m** [comment]
提供最小复现看看。

**Comment by 1353823765** [comment]
在config中配置dva:{},就可以了，可以用useselector代替connect
